test/models/post_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
+  def setup
+    @user = users(:musa)
+    @post = Post.new(body: 'Lorem ipsum', user_id: @user.id)
+  end
+
+  test 'should be valid' do
+    assert @post.valid?
+  end
+
+  test 'user id should be present' do
```

Comment:
We can safely remove testing for this as the association test below handles the presence of user_id